data/Language/antlr4.yaml

```diff
@@ -0,0 +1,36 @@
+identifier: antlr4
+creation_date: 2017
+wikidata: Q295801
+full_name: Another Tool For Language Recognition
+grammar_file: https://github.com/antlr/grammars-v4/tree/master/antlr4
+aliases:
+  - antlr4
+extensions:
+  - g4
+line_continuation:
+  - backslash
+delimiters:
```

Comment:
I suppose we have a rule delimiter `;` as well. And lines are assumed to continue over any whitespace unless a rule delimiter is encountered. 
example rule:
```
genericAssocList
    :   genericAssociation
    |   genericAssocList ',' genericAssociation
    ;
```

Although I haven't seen the statement delimiter mentioned in most of the languages, probably a useful thing to include ? (most new languages use a semicolon, but some older ones use newlines / full stops as well)

Reply:
Seems practical, will add this too.